pkg/cloudprovider/vsphere/instances.go

```diff
@@ -130,14 +131,20 @@ func (i *instances) InstanceID(ctx context.Context, nodeName types.NodeName) (st
 // InstanceType returns the type of the instance identified by name.
 func (i *instances) InstanceType(ctx context.Context, name types.NodeName) (string, error) {
 	klog.V(4).Info("instances.InstanceType() called")
-	return i.nodeManager.nodeNameMap[string(name)].NodeType, nil
+	if nodeInfo, ok := i.nodeManager.nodeNameMap[string(name)]; ok {
+		return nodeInfo.NodeType, nil
+	}
+	return "", fmt.Errorf("cannot find node with nodeName %s in nodeUUIDMap", name)
```

Comment:
@andrewsykim I added the error message, PTAL :)

Reply:
minor typo in this error message, should be nodeNameMap, not nodeUUIDMap